env-defaults

```diff
@@ -275,6 +275,14 @@ SDS_MASKFILL_REQUESTS_MEMORY=128Mi
 SDS_MASKFILL_LIMITS_CPU=128m
 SDS_MASKFILL_LIMITS_MEMORY=512Mi
 
+TRAJECTORY_SUBSETTER_IMAGE=sds/trajectory-subsetter:latest
+TRAJECTORY_SUBSETTER_IMAGE_PULL_POLICY='IfNotPresent'
+TRAJECTORY_SUBSETTER_PARALLELISM=2
+TRAJECTORY_SUBSETTER_REQUESTS_CPU=128m
+TRAJECTORY_SUBSETTER_REQUESTS_MEMORY=128Mi
+TRAJECTORY_SUBSETTER_LIMITS_CPU=128m
+TRAJECTORY_SUBSETTER_LIMITS_MEMORY=512Mi
```

Comment:
I'm impressed the subsetter worked for a >600 MB granule with 512MB of memory for the service. Does 512MB seem like enough memory to you?

Reply:
Preserving offline conversation: Chris ran a request in the Sandbox environment. Even though the granule was > 600 MB, he saw the pod memory usage max out at 216MB.